tests/conftest.py

```diff
@@ -74,8 +75,8 @@ def client():
                    ],
                    "inLanguage": "en",
                    "tags": "weather, uk, 2011, temperature, humidity",
-                   "price": 50
-
+                   "price": 50,
+                   "type": "Dataset"
```

Comment:
I suggest having the type always in lower case as is included in the oep-08 spec